docs/stack/subgraph/queries.md

```diff
@@ -0,0 +1,346 @@
+
+*Examples*
+<details>
+  <summary> Details of all `daos` indexed by the DAOstack subgraph </summary>
+
+  <body>
+```
+query {
+  daos{
+    name
+    id
+    reputationHoldersCount
+    proposals{
+      id
+      title
+    }
+  }
+}
+```
+  </body>
+</details>
+
+<details>
+  <summary> Details of all `Reputation Holders` in DAOstack DAOs </summary>
+
+  <body>
+```
+query {
+  reputationHolders {
+    id
+    address
+    balance
+    dao{
+      name
+    }
+  }
+}
+```
+  </body>
+</details>
+
+### Filter by fields
+
+To query for a subset of Entities you can add `where: {}` parameter to filter for different properties. You can filter for single or multiple properties.
+
+*Examples*
+
+<details>
+  <summary> To get all proposals submitted on 2019 Halloween, we can filter for the time interval on `createdAt` property </summary>
+
+  <body>
+```
+query {
+  proposals (
+    where : {
+      createdAt_gt: 1572480000,
+      createdAt_lt: 1572566400
+    }
+  ) {
+    id
+  	title
+    dao {
+      name
+    }
+  }
+}
+```
+  </body>
+</details>
+
+<details>
+  <summary> Get all `daos` with more than 200 reputation holders </summary>
+
+  <body>
+```
+query {
+  daos (
+    where: {
+      reputationHoldersCount_gt: 200
+  }) {
+    name
+    reputationHoldersCount
+  }
+}
+```
+  </body>
+</details>
+
+<details>
+  <summary> Genesis DAO proposals that contains word 'Reputation' in title
+  </summary>
+
+  <body>
+```
+query{
+  proposals(
+    where: {
+      dao: "0x294f999356ed03347c7a23bcbcf8d33fa41dc830"
+      title_contains: "Reputation"
+    }
+  ){
+    title
+    dao{
+      name
+    }
+  }
+}
+```
+  </body>
+</details>
+
+NOTE:
+  
+  - The suffix `_contains` in the above example is used for the comparison
+  - Some suffixes are only supported for specific types. For example, Boolean only supports `_not`, `_in`, and `_not_in`.
+  - <details>
+    <summary> Complete list of suffix is </summary>
+    - _not
+    - _gt
+    - _lt
+    - _gte
+    - _lte
+    - _in
+    - _not_in
+    - _contains
+    - _not_contains
+    - _starts_with
+    - _ends_with
+    - _not_starts_with
+    - _not_ends_with
+  </details>
+
+### Sort by field values
+
+*Examples*
+
+To query for a sorted list you can add `orderBy` parameter to sort by a specific property. Also, you can specify the direction of sort `asc` for ascending and `desc` for descending.
+
+<details>
+  <summary> Sort Reputation Holders by their reputation balance</summary>
+
+  <body>
+```
+query {
+  reputationHolders(
+    orderBy: balance,
+    orderDirection: desc
+  ){
+    address
+    balance
+  }
+}
+```
+  </body>
+</details>
+
+<details>
+  <summary> Sort DAOs by number of boosted proposals it has </summary>
+
+  <body>
+```
+query {
+  daos (
+    orderBy: numberOfBoostedProposals,
+    orderDirection: asc
+  ) {
+    name
+    numberOfBoostedProposals
+  }
+}
+```
+  </body>
+</details>
+
+### Paginate
```

Comment:
It's worth mentioning here that there is a 1000 entities limit per query.

Reply:
Ohh yeah... I found that out just yesterday. 
Will add that too